server/command.go

```diff
@@ -323,3 +324,65 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 	p.postCommandResponse(args, fmt.Sprintf("Unknown action %v", action))
 	return &model.CommandResponse{}, nil
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	github := model.NewAutocompleteData("github", "[command]", "Available commands: connect, disconnect, todo, subscribe, unsubscribe, me, settings")
+
+	connect := model.NewAutocompleteData("connect", "", "Connect your Mattermost account to your GitHub account")
+	private := model.NewAutocompleteData("private", "", "If used, the github bot will ask for read access to your private repositories")
+	connect.AddCommand(private)
+	github.AddCommand(connect)
+
+	disconnect := model.NewAutocompleteData("disconnect", "", "Disconnect your Mattermost account from your GitHub account")
+	github.AddCommand(disconnect)
+
+	todo := model.NewAutocompleteData("todo", "", "Get a list of unread messages and pull requests awaiting your review")
+	github.AddCommand(todo)
+
+	subscribe := model.NewAutocompleteData("subscribe", "[owner/repo] [features] [flags]", "Subscribe the current channel to receive notifications about opened pull requests and issues for an organization or repository. [features] and [flags] are optional arguments")
+	subscribe.AddTextArgument("Owner/repo to subscribe to", "[owner/repo]", "")
+	subscribe.AddTextArgument("(Optional) Comma-delimited list of one or more of: issues, pulls, pushes, creates, deletes, issue_comments, pull_reviews, label:\"<labelname>\". Detaults to pulls,issues,creates,deletes", "[features]", `/[^,-\s]+(,[^,-\s]+)*/`)
+	flags := []model.AutocompleteListItem{{
+		HelpText: "events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)",
+		Hint:     "(optional)",
+		Item:     "--exclude-org-member",
+	}}
+	subscribe.AddStaticListArgument("Currently supports --exclude-org-member", false, flags)
+
+	github.AddCommand(subscribe)
+
+	subscribeList := model.NewAutocompleteData("subscribe list", "", "List the current channel subscriptions")
```

Comment:
When I type `/github subscribe list` and then press `enter`, `[feature]` is shown as suggested input.
![Screenshot from 2020-06-15 11-24-14](https://user-images.githubusercontent.com/16541325/84641053-d2ce1680-aefa-11ea-9436-c73bf64e2050.png)

@iomodo Could this be a bug in Mattermost?

Reply:
I mention this in the PR notes- because `/github subscribe [owner/repo] [features]` is its own command, the autocomplete parses `/github subscribe list` as the subscribe command with `list` as the owner/repo field.

Reply:
Not a bug, for autocomplete there is no way to detect the difference. Clean fix will be to use `/github subscriptions` command to list them.